volume/local/local_unix.go

```diff
@@ -85,3 +88,12 @@ func (v *localVolume) mount() error {
 	err := mount.Mount(v.opts.MountDevice, v.path, v.opts.MountType, mountOpts)
 	return errors.Wrapf(err, "error while mounting volume with options: %s", v.opts)
 }
+
+func (v *localVolume) CreatedAt() (time.Time, error) {
+	fileInfo, err := os.Stat(v.path)
```

Comment:
A stat is not that expensive and its much better to have one source of truth than trying to rely on in-memory state.